packages/ngtools/webpack/src/transformers/remove-ivy-dev-calls_spec.ts

```diff
@@ -0,0 +1,115 @@
+/**
+ * @license
+ * Copyright Google Inc. All Rights Reserved.
+ *
+ * Use of this source code is governed by an MIT-style license that can be
+ * found in the LICENSE file at https://angular.io/license
+ */
+import { tags } from '@angular-devkit/core';  // tslint:disable-line:no-implicit-dependencies
+import { transformTypescript } from './ast_helpers';
+import { removeIvyDevCalls } from './remove-ivy-dev-calls';
+
+const input = tags.stripIndent`
+  export class AppModule {
```

Comment:
Are the `i0` imports elided too? I think our code would remove them, but they aren't present in the input so we don't test that here.

I think webpack might remove them? But I'm not 100% sure, because by the time webpack sees it's just a static import. No other tooling sees the `i0` and knows it was being used but isn't needed anymore.

Reply:
Now that I think about it more, I'm not sure this matters. I **think** other `i0` calls will still remain here.